SECURITY.md

```diff
@@ -0,0 +1,23 @@
+<h2>Responsible disclosure policy</h2>
+
+At the IOTA Foundation, we consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our security team:
```

Comment:
```suggestion
At the IOTA Foundation, we consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability in Trinity, please follow the guidelines below to report it to our security team:
```

Added clarification since Trinity's disclosure process is different from other IF projects (vulnerabilities are submitted to Bugcrowd instead of security@iota.org)